gunicorn/workers/_gaiohttp.py

```diff
@@ -16,6 +22,12 @@ class AiohttpWorker(base.Worker):
 
     def __init__(self, *args, **kw):  # pragma: no cover
         super().__init__(*args, **kw)
+        cfg = self.cfg
+        if cfg.ssl_version:
```

Comment:
it should be `self.cfg.is_ssl` there, `cfg.ssl_version` is always set. (default is [ssl.PROTOCOL_TLSv1](https://github.com/benoitc/gunicorn/blob/master/gunicorn/config.py#L1642) .

Also why not wrapping the socket directly?